code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -1760,3 +1760,67 @@
 		H.vomit(10)
 	..()
 	return TRUE // update health at end of tick
+
+/datum/reagent/medicine/neurostim
+	name = "neuro stimulant"
+	description = "A complex mixture of neuro activating chemicals and a fresh dose of a specific being DNA. the DNA seems to be that of a goat. a highly unethical mixture likely made from stem cells and other nasty bits, THERE IS ABSOLUTELY FEV IN THIS."
+	color = "#1AF708"
+	metabolization_rate = 1.0 * REAGENTS_METABOLISM
+	ghoulfriendly = TRUE
+
+/datum/reagent/medicine/neurostim/on_mob_life(mob/living/carbon/M)
+	ADD_TRAIT(M, TRAIT_SURGERY_HIGH, src)
+	ADD_TRAIT(M, TRAIT_UNETHICAL_PRACTITIONER, src)
+	ADD_TRAIT(M, TRAIT_UNETHICAL_PRACTITIONER_BRAINWASHING, src)
+	ADD_TRAIT(M, TRAIT_CYBERNETICIST, src)
+	ADD_TRAIT(M, TRAIT_CYBERNETICIST_EXPERT, src)
+	ADD_TRAIT(M, TRAIT_CHEMWHIZ, src)
+	ADD_TRAIT(M, TRAIT_MEDICALEXPERT, src)
+	ADD_TRAIT(M, TRAIT_MEDICALGRADUATE, src)
+	M.Jitter(2)
+	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, -5)
```

Comment:
What's the point of having this healing? You're presumably going to be taking it every round when you join, so you wouldn't have any brain damage to heal.

Reply:
its fluff. its not ment to do anything its entirely on the principle that its creating new neuro pathways.....im aware that it will never actually heal anything

Reply:
But it does trigger health and icon updates, which aren't exactly cheap.